<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatClient</class>
 <widget class="QMainWindow" name="ChatClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="text_feed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>591</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="send_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IP_line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.83</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PORT_line">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>60000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
